packages/program-boilerplate/package.json

```diff
@@ -17,10 +17,14 @@
     "@babel/cli": "^7.1.5",
     "@babel/core": "^7.1.5",
     "@babel/preset-env": "^7.1.5",
-    "np": "^2.18.3"
+    "@types/express": "^4.17.0",
+    "@types/node": "^12.0.8",
+    "np": "^2.18.3",
+    "typescript": "^3.5.1"
   },
   "dependencies": {
     "@babel/polyfill": "^7.0.0",
```

Comment:
I don't think this is used or required anymore since we are on a newer javascript runtime.